import { main } from "./main"

main().catch((error) => {
  throw error
})
